RecoTracker/LSTCore/interface/alpaka/Constants.h

```diff
@@ -11,7 +11,7 @@
 
 namespace lst {
```

Comment:
would it make sense to put everything to `namespace ALPAKA_ACCELERATOR_NAMESPACE {` ?

Reply:
Yeah, I think that makes sense